modules/meta.analysis/DESCRIPTION

```diff
@@ -25,19 +25,19 @@ Description: The Predictive Ecosystem Carbon Analyzer (PEcAn) is a scientific
     streamline the interaction between data and models, and to improve the
     efficacy of scientific investigation. The PEcAn.MA package contains
     the functions used in the Bayesian meta-analysis of trait data.
-Depends:
+Imports:
+    coda (>= 0.18),
     XML,
+    R2WinBUGS,
```

Comment:
We don't actually use any functionality from the R2WinBUGS package in PEcAn.MA, we only hyperlink to it in the documentation as an acknowledgement of design inspiration. 

You're totally correct that the R checks demand that we declare dependencies on all the packages whose documentation we link to, but in this case I think the correct fix is to avoid linking to it rather than to declare the dependency -- PEcAn users don't need to have R2WinBUGS installed to understand that we were inspired by it. 

Can you please remove this line and then rewrite the docs for `write.ma.model` to cite R2WinBUGS in plain text instead of with a link? Like so:

```
- ##'  Writes a meta-analysis model based on available data and prior specification. Adapted from \code{\link[R2WinBUGS]{write.model}}
+ ##'  Writes a meta-analysis model based on available data and prior specification. Adapted from \code{R2WinBUGS::write.model}
```